Extensions/retags.js

```diff
@@ -68,26 +68,27 @@ XKit.extensions.retags = {
 				return false;
 			}
 			// popover
-			if ($t.hasClass('note')) {
+			if ($t.hasClass('note-item')) {
 				cls = 'with_commentary';
 				$c = $t;
-				url = $c.find('.action').data('post-url');
+				var peepr_data = $c.data('peepr');
+				url = 'http://' + peepr_data.tumblelog.trim() + '/post/' + peepr_data.postId.trim();
 			// Activity
 			} else if ($t.hasClass('ui_note')) {
 				if ($t.find('.part_response').length) {
 					$t.addClass('is_response');
 				}
 				cls = 'is_retags';
-				$c = $t.find('.stage');
-				url = $c.find('.part_glass').attr('href');
+				$c = $t.find('.stage > div:last-child').last();
+				url = $t.find('.part_glass').attr('href');
 			// dashboard
 			} else if ($t.hasClass('notification') && !XKit.browser().mobile) {
-				$c = $t.find('.notification_sentence');
+				$c = $t.find('.notification_sentence').last();
```

Comment:
Because (see below line 185) we're now pasting the retags content after `$c` instead of appending within it. (Flexbox style weirdness was making the append not work very well with the new design.)